test/unit/specs/components/ResultsPage.spec.js

```diff
@@ -1,9 +1,26 @@
-import { shallow } from '@vue/test-utils';
+import Vuex from 'vuex';
+import { shallow, createLocalVue } from '@vue/test-utils';
+
 import ResultsPage from '@/components/ResultsPage';
+import exampleResponses from '../../example-responses';
+
```

Comment:
It appears that using `createLocalVue` is required when testing components that interact with the store, at least if you're using `mapState` to avoid having to write all the computed properties yourself.

Tests are based on documentation found here:

https://vue-test-utils.vuejs.org/en/guides/using-with-vuex.html